seeds/create_users.js

```diff
@@ -1,10 +1,14 @@
+const crypto = require("crypto");
```

Comment:
import statement for consistency

Reply:
oh, this is a seed file... no worries if it's a different ecma or whatever.

Reply:
I don't think you can use imports here because these files aren't run after transpile

Reply:
yeah, makes sense.